index.html

```diff
@@ -33,6 +33,6 @@ <h5 class="modal-title" id="exampleModalLabel">Success!</h5>
     </div>
 
     <script src="node_modules/jquery/dist/jquery.slim.min.js"></script>
```

Comment:
there is no need to import jQuery here, it's already imported in your bundle because Bootstrap import jQuery